pkg/types/types.go

```diff
@@ -44,6 +47,9 @@ type SmartState interface {
 	NewestAddrByAlias(alias proto.Alias) (proto.WavesAddress, error)
 	NewestLeasingInfo(id crypto.Digest) (*proto.LeaseInfo, error)
 	IsStateUntouched(account proto.Recipient) (bool, error)
+	AssetInfoByID(id proto.AssetID, filter bool) (*proto.AssetInfo, error)
```

Comment:
Remove this method, use internal methods instead

Reply:
Removed